### DISCUSSION TITLE
coc-references: current line color is broken after update, how to fix?

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [neoclide/coc.nvim#5055] [Q&A] [answered]
As you see on the screenshot - line is white, with unreadable text:

![image](https://github.com/neoclide/coc.nvim/assets/3514015/c02e5d26-ac49-4c54-ab48-93cb6f15b2b3)

Thanks...

**Comment by fannheyward** [comment]
The hl group should be `CocLocationLine`, coc.nvim didn't change this recently, what do you mean `after update`?

**Reply by vitaly-zdanevich** [reply]
> what do you mean after update?

Just I remember that previously it was ok, maybe I did something with my configs, but I remember that recently I updated my Vim plugins.

**Reply by vitaly-zdanevich** [reply]
I switched to `v0.0.82` - now colors are ok.

**Reply by vitaly-zdanevich** [reply]
But still, I want to use the current git version - how to change selected line color? `CocLocationLine` does not affect the color of the selection line :(

**Comment by chemzqm** [comment, marked as answer]
Use a better color scheme or change highlight for `CocListLine`

**Reply by vitaly-zdanevich** [reply]
E411: Highlight group not found: CocListCurrent

Current Coc version after `git pull`

**Reply by chemzqm** [reply]
Sorry, `CocListLine`
